Tools/SandboxTest.ps1

```diff
@@ -207,6 +207,11 @@ Write-Host @'
 '@
 Update-EnvironmentVariables
 
+Write-Host @'
+
+--> Opening ARP Entries
+'@
+appwiz.cpl

```

Comment:
```suggestion
winget list --accept-source-agreements
```
It exposes most of the same info at the command line, and if the ID doesn't appear as the user thinks it should they can investigate? I'm just not a big fan of opening windows I can't pipe to a file :) 

The other way to do this is via PowerShell so that you can JSONify and analyze the info outside the sandbox for _fully automated luxury ARP verification_ but it's not a one liner.

Reply:
Hmm. I guess that would work too. I think that having the full ARP verification would be helpful, I'll take a look at that

Reply:
My idea for doing that was to get the new ARP entries, JSONify them, dump them to a text file which is accessible outside the sandbox, and then on the outside, `ConvertFrom-JSON` and see if you can match them to the `PackageName`, `Publisher`, and `PackageVersion` in the manifest (excluding MSIX, which you'd have to do via `Get-AppxPackage` 🤢).

This is way more complicated than your one liner, so I understand if you don't want to do that with this (i.e. I'll approve this and take a look at doing that sometime myself).

Reply:
My thought with this was more that it sucks to have to open the ARP table every time I use the sandbox. I think full validation would be great, but we may also want to be careful about adding too much automation. Part of the beauty of the sandbox script is that it allows for manual validation of the installation and the ARP entries, which is a key part of moderation IMO

Reply:
Yeah, in my patched version of the script `winget list` was the first thing I added (never got around to upstreaming the change), totally understandable.

I think we need to write some automated client-side (i.e. not in the validation pipeline) ARP validation purely for peace-of-mind (sometimes you're going fast and think you remember how the version numbers for that package looked, which isn't a good habit but it happens with the momentum of the repo). I'll approve this then.

Reply:
I think I got it showing the new entries now

Reply:
Ugh. That only works if its the only package that gets installed. Time to do more digging